### PR TITLE
Refactor click

### PR DESCRIPTION
**Pull request by myakove**
Change the way that we get addon/operator args.
We get it as dict which is easier to work on.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,14 +39,28 @@ poetry run python app/cli.py --help
 
 Each command can be run via container `podman run quay.io/redhat_msi/ocp-addons-operators-cli` or via poetry command `poetry run app/cli.py`
 
+### Addon/Operator user args
+
+Each `--addon` or `operator` accept args, the format is `arg=value;`
+###### Common args:
+* `name=name`: Name of the operator/addon to install/uninstall
+* `timeout=300`: timeout to wait for the operator/addon to be installed/uninstalled
```
Comment (rnetser) [inline README.md:47]:
```suggestion
* `timeout=300`: timeout in seconds to wait for the operator/addon to be installed/uninstalled
```

File: README.md [hunk 2]
```diff
@@ -39,14 +39,28 @@ poetry run python app/cli.py --help
...
+
+###### Addon args:
+* `rosa=true`: Use rosa cli to install/uninstall the addon
+
+###### Operator args:
+* `iib=/path/to/iib:123456`: Install the operator using the provided IIB
```
Comment (rnetser) [inline README.md:53]: what about ns, target-namespaces etc?

File: README.md [hunk 3]
```diff
@@ -39,14 +39,28 @@ poetry run python app/cli.py --help
...
+
+
 #### Install Addon
 ##### One addon
 
 ```
 podman run quay.io/redhat_msi/ocp-addons-operators-cli \
     addon \
```
Comment (rnetser) [inline README.md:61]:
```suggestion
    addons \
```

File: README.md [hunk 4]
```diff
@@ -126,10 +136,20 @@ To run multiple operators install in parallel pass -p,--parallel.
 
 ```
 podman run quay.io/redhat_msi/ocp-addons-operators-cli \
-    operator \
+    operators \
+    --kubeconfig ~/work/CSPI/kubeconfig/rosa-myk412 \
+    -o 'name=rhods-operator;namespace=redhat-ods-operator' \
```
Comment (rnetser) [inline README.md:141]:
```suggestion
    -o 'name=rhods-operator;namespace=redhat-ods-operator;timeout=600' \
```

File: README.md [hunk 5]
```diff
@@ -150,9 +170,9 @@ To run multiple operators uninstall in parallel pass -p,--parallel.
 
 ```
 podman run quay.io/redhat_msi/ocp-addons-operators-cli \
-    operator \
+    operators \
     --kubeconfig ~/work/CSPI/kubeconfig/rosa-myk412 \
-    -o 'rhods-operator|namespace=redhat-ods-operator' \
-    -o 'servicemeshoperator' \
+    -o 'name=rhods-operator;namespace=redhat-ods-operator' \
+    -o 'name=servicemeshoperator' \
```
Comment (rnetser) [inline README.md:176]:
```suggestion
    -o 'name=servicemeshoperator;timeout=600' \
```

**Review by rnetser — CHANGES_REQUESTED** [review]

**Review by oharan2 — APPROVED** [review]

**Pull request merged**